main

```diff
@@ -0,0 +1,49 @@
+require './app'
+
+def main
+  SchoolLibraryApp.new.run
+end
+
+class SchoolLibraryApp
+  def initialize
+    puts 'Welcome to School Library OOP App!'
+  end
+
+  def menu
+    puts
+    puts 'Please choose an option by entering a number'
+    @list = {
+      1 => 'List all books',
+      2 => 'List all people',
+      3 => 'Create a person',
+      4 => 'Create a book',
+      5 => 'Create a rental',
+      6 => 'List all rentals for a given person id',
+      7 => 'Exit'
+    }
+    @list.each { |id, option| puts "#{id}: #{option}" }
+    gets.chomp.to_i
+  end
```

Comment:
- [OPTIONAL] Great job with awesome UI design. I would recommend you to make your app more user-friendly by displaying a message that informs the user who selects option 1 or 2 when no book or no person is available at that moment rather than just keep telling the user to select an option which he or she has done couple of times. You're app is works perfectly. Keep up the good work. :tada: